web/app.js

```diff
@@ -774,6 +776,23 @@ const PDFViewerApplication = {
     );
   },
 
+  get supportsCaretBrowsingMode() {
+    return shadow(
+      this,
+      "supportsCaretBrowsingMode",
+      AppOptions.get("supportsCaretBrowsingMode")
+    );
+  },
+
+  moveCaret(isUp, select) {
+    this._caretBrowsing ||= new CaretBrowsingMode(
+      this.appConfig.mainContainer,
+      this.appConfig.viewerContainer,
+      this.appConfig.toolbar.container
```

Comment:
Elsewhere throughout this file the code is written to handle the `toolbar` not being defined, hence:
```suggestion
      this.appConfig.toolbar?.container
```